src/client/test-run/iframe.js.mustache

```diff
@@ -1,10 +1,10 @@
 (function () {
     var IframeDriver = window['%testCafeIframeDriver%'];
     var driver       = new IframeDriver({{{testRunId}}}, {
-        selectorTimeout: {{{selectorTimeout}}},
-        pageLoadTimeout: {{{pageLoadTimeout}}},
-        dialogHandler:   {{{dialogHandler}}},
-        speed:           {{{speed}}}
+        selectorTimeout:    {{{selectorTimeout}}},
```

Comment:
You added additional indents here and in `src/client/test-run/index.js.mustache`